components/ui/GraphList/GraphList.tsx

```diff
@@ -0,0 +1,80 @@
+'use-client'
+
+import Graph from '@/components/ui/Graph/Graph';
+import React from 'react';
+
+const GraphList: React.FC = () => {
+  const graphConfig = {
+    labels: ['Total Leads', 'Hot Deals', 'Executive Checked-in'],
+    counts: [10, 20, 30],
+    colors: ['#5630FF', '#FF7B7B', '#07BEAA'],
+    chartType: 'line',
+  };
+
+  const graphData = [
+    { month: 'January', sales: 100 },
+    { month: 'February', sales: 150 },
+    { month: 'March', sales: 125 },
+    { month: 'April', sales: 150 },
+    { month: 'May', sales: 160 },
+    { month: 'June', sales: 140 },
+    { month: 'July', sales: 170 },
+    { month: 'August', sales: 150 },
+    { month: 'September', sales: 130 },
+    { month: 'October', sales: 180 },
+    { month: 'November', sales: 170 },
+    { month: 'December', sales: 200 },
+  ];
+
+  return (
+    <div className='bg-[#F7F7FB]'>
+      this is dashboard page
```

Comment:
I believe this text ('this is dashboard page') should not appear into this component level.